src/main/java/org/jabref/model/util/DirectoryMonitorManager.java

```diff
@@ -0,0 +1,32 @@
+package org.jabref.model.util;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.io.monitor.FileAlterationListener;
+import org.apache.commons.io.monitor.FileAlterationObserver;
+
+public class DirectoryMonitorManager {
+
+    private final DirectoryMonitor directoryMonitor;
+    private final List<FileAlterationObserver> observers = new ArrayList<>();
+
+    public DirectoryMonitorManager(DirectoryMonitor directoryMonitor) {
+        this.directoryMonitor = directoryMonitor;
+    }
+
+    public void addObserver(FileAlterationObserver observer, FileAlterationListener listener) {
+        directoryMonitor.addObserver(observer, listener);
+        observers.add(observer);
+    }
+
+    public void removeObserver(FileAlterationObserver observer) {
+        directoryMonitor.removeObserver(observer);
+        observers.remove(observer);
+    }
+
+    public void unregister() {
```

Comment:
Here a JavaDoc comment could help (if you want to write JavaDoc comments). No comments are also OK.